onnx/defs/generator/old.cc

```diff
@@ -101,57 +102,62 @@ ONNX_OPERATOR_SET_SCHEMA(
 
           if (nullptr != value_int) {
             // OpSchema::Verify check ensures that the attribute value has_i():
-            if (!value_int->has_i())
+            if (!value_int->has_i()) {
               fail_shape_inference("Attribute 'value_int' expect an integer.")
-                  updateOutputElemType(ctx, 0, TensorProto::INT64);
```

Comment:
I realize this is a pre-existing problem, but minor nit: better to add semicolon after fail_shape_inference(...) here and down below